src/test/pmemwrite/write.c

```diff
@@ -114,6 +114,8 @@ pmemwrite_blk(struct pmemwrite *pwp)
 
 	int i;
 	int ret = 0;
+	size_t blksize = pmemblk_bsize(pbp);
+	char *blk = malloc(blksize);
```

Comment:
Okay, so we decided you can't use the `MALLOC()` macro here, so checking for `NULL` back from `malloc` must be done explicitly.